internal/common/validate/cluster_error_checks.go

```diff
@@ -0,0 +1,8 @@
+package validate
+
+import admin20240530 "go.mongodb.org/atlas-sdk/v20240530005/admin"
+
+// ErrorClusterIsAsymmetrics must be used with the v20240530005 admin.
+func ErrorClusterIsAsymmetrics(err error) bool {
+	return err != nil && admin20240530.IsErrorCode(err, "ASYMMETRIC_SHARD_UNSUPPORTED")
```

Comment:
```suggestion
	return admin20240530.IsErrorCode(err, "ASYMMETRIC_SHARD_UNSUPPORTED")
```
no need to check err nilness

Reply:
Thanks. Updated in 84a2beffae4f6f942b585587db489dc57deb3910